silk/sql.py

```diff
@@ -77,7 +80,13 @@ def execute_sql(self, *args, **kwargs):
             return iter([])
         else:
             return
-    sql_query = q % tuple(force_str(param) for param in params)
+    str_params = []
+    for param in params:
+        try:
+            str_params.append(force_str(param))
+        except DjangoUnicodeDecodeError:
```

Comment:
`DjangoUnicodeDecodeError` is a trivial subclass of `UnicodeDecodeError` and you're not doing anything with the exception so I think it would be easier to skip the import and just catch `UnicodeDecodeError`.

Ref: https://github.com/django/django/blob/45ba7683a6e399815731a13cceb3acb518a2f1f2/django/utils/encoding.py#L11